include/odp/api/spec/packet_io.h

```diff
@@ -412,6 +412,9 @@ typedef union odp_pktout_config_opt_t {
 		/** Insert SCTP checksum on packet by default */
 		uint64_t sctp_chksum     : 1;
 
+		/** Packet fast free */
+		uint64_t fast_free       : 1;
```

Comment:
In my opinion this should not be called fast_free but something more descriptive (packet_references_enabled?).

If someone comes up with another flag like this, indicating, say, that there are no multi-segment packets, we do not want to add a flag called fast_free2 or something,

Reply:
Ack, will change it to 'packet_refs_enabled'